api-template-java/src/main/java/com/slf/services/Handler.java

```diff
@@ -49,9 +42,20 @@
 	// @Override
 	public Response handleRequest(Request input, Context context) {
 
+		// Initialize the environment specific variables. Each environment has different properties files to store env specific data.
+		// For example, for development environment specific details, add configurations as key,value pairs in 'dev.properties'
```

Comment:
What if I don't use any configuration value? This code should be available as a template but commented out.

Reply:
Done.